.golangci.yml

```diff
@@ -129,5 +129,9 @@ issues:
       linters:
         - gocritic
         - dupl
+    - path: cli
+      linters:
+        - forbidigo
+        - contextcheck
```

Comment:
Can we check if we still need to skip running `contextcheck` for cli package ?

Reply:
Yes we do, as now we are not passing context anywhere, rather using `command.Context`.